src/main.c

```diff
@@ -602,6 +602,75 @@ void handleSign(uint8_t p1,
             ux_flow_display(APPROVAL_UNFREEZEASSET_TRANSACTION,
                             ((txContent.dataBytes > 0) ? true : false));
 
+            break;
+        case FREEZEBALANCEV2CONTRACT: // Freeze TRX
+            if (txContent.resource == 0)
+                strcpy(fullContract, "Bandwidth");
+            else
+                strcpy(fullContract, "Energy");
+
+            print_amount(txContent.amount[0], (char *) G_io_apdu_buffer, 100, SUN_DIG);
+            getBase58FromAddress(txContent.account,
+                (uint8_t *)toAddress, &sha2, HAS_SETTING(S_TRUNCATE_ADDRESS));
+
+            ux_flow_display(APPROVAL_FREEZEASSETV2_TRANSACTION,
+                            ((txContent.dataBytes>0)? true : false));
+            break;
+        case UNFREEZEBALANCEV2CONTRACT: // unreeze TRX
+            if (txContent.resource == 0)
+                strcpy(fullContract, "Bandwidth");
+            else
+                strcpy(fullContract, "Energy");
+
+            print_amount(txContent.amount[0], (char *) G_io_apdu_buffer, 100, SUN_DIG);
```

Comment:
@ClarkChenc Will have to run clang-format on your files with : 

 `clang-format-11 -i $(find src -type f \( -name '*.c' -o -name '*.h' \))`

so that they are compliant with the lint rules of the CI.

Reply:
files have been formatted, thanks a lot.